server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/Task.java

```diff
@@ -33,7 +33,7 @@ public interface Task<RESULT> {
     /**
      * Start current task. This method will be called by {@link TaskExecutor} for fire a task
      */
-    void start(JobContext context);
+    void start(JobContext context, TaskContext taskContext);
```

Comment:
two context make it more complex, can we keep one context only?

Reply:
it has been correct as make jobContext as a member of taskContext